src/irule.cpp

```diff
@@ -320,13 +323,28 @@ main( int argc, char **argv ) {
                 }
 
                 if ( rulegen ) {
+                    constexpr auto null_term{1};
                     if ( startsWith( buf, "INPUT" ) || startsWith( buf, "input" ) ) {
                         gotRule = 1;
-                        trimSpaces( trimPrefix( buf ) );
+                        constexpr auto skip_len{5};
+                        const auto copy_length{std::strlen(buf) + null_term - skip_len};
+
+                        // Blank out "INPUT"
+                        std::memmove(buf, buf + skip_len, copy_length);
+
+                        auto res{boost::algorithm::trim_left_copy(std::string{buf})};
+                        std::strncpy(buf, res.c_str(), res.size() + null_term);
```

Comment:
This looks identical to the body of the other if-stmt. Perhaps we can put it in a function?

Reply:
Looks like this has been addressed

Reply:
Resolving.